src/utils.py

```diff
@@ -103,4 +103,4 @@ def return_random_number() -> int:
     float
     '''
 
-    return np.random.randint(0, 100)
```

Comment:
why, it is happning?

Reply:
I think because the previous line was replaced with an entire new line which is an exact copy of the previous line.